public/docs/js/latest/guide/forms.jade

```diff
@@ -283,6 +306,8 @@ figure.image-display
   The diagnostic is evidence that we really are flowing values from the input box to the model and
   back again. **That's two-way data binding!**
 
+  Notice that we also added a `name` attribute to our `<input>` tag. This is a requirement when using `[(ngModel)]` in combination with a form. 
```

Comment:
That's a good idea. We make that point later in the article, but it probably belongs here as well